content/en/docs/tasks/administer-cluster/manage-resources/cpu-default-namespace.md

```diff
@@ -44,12 +57,12 @@ Create the LimitRange in the default-cpu-example namespace:
 kubectl apply -f https://k8s.io/examples/admin/resource/cpu-defaults.yaml --namespace=default-cpu-example
 ```
 
-Now if a Container is created in the default-cpu-example namespace, and the
-Container does not specify its own values for CPU request and CPU limit,
-the Container is given a default CPU request of 0.5 and a default
+Now if you create a Pod in the default-cpu-example namespace, and any container
```

Comment:
As previously asked, does the default-cpu-example apply on a per-pod or per-container basis? I'm thinking the two should match in that sentence.